node/src/components/rpc_server/rpcs/docs.rs

```diff
@@ -100,6 +100,7 @@ pub(crate) static OPEN_RPC_SCHEMA: Lazy<OpenRpcSchema> = Lazy::new(|| {
     schema.push_with_optional_params::<GetAuctionInfo>(
         "returns the bids and validators as of either a specific block (by height or hash), or the most recently added block",
     );
+    schema.push_with_optional_params::<GetEraSummary>("returns the era summary at either a specific block (by height or hash), or the most recently added block");
```

Comment:
We'll need to update the description here if we change to use `GlobalStateIdentifier`.

Reply:
Will deal with this as agreed offline

Reply:
As discussed, we will stick with BlockIdentifier because we can't go from `GlobalStateIdentifier::StateRootHash` to a get a `Block` to fill `EraSummary.block_hash`  as state root hashes aren't 1:1 with blocks - if a block does not execute anything then it will carry forward previous state root hash. @darthsiroftardis and @EdHastingsCasperLabs agrees on that